source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,150 @@
+* Angle bracket invocation for built-in components
+* Support for native decorators
+
+[3.11](https://blog.emberjs.com/2019/07/15/ember-3-11-released.html)
+* Forwarding Element Modifiers with `...attributes`
+* The `{{fn}}` and `{{on}}` modifiers
+
+This release kicks off the 3.14 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.13
+
+Ember.js 3.13 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are five (5) new features, one (1) deprecation, and several bugfixes in this version.
+
+#### New Features (5)
+
+**Tracked Properties and Tracked Property Updates (1 of 5)**
+
+Tracked properties introduce a simpler and more ergonomic system for tracking state change in Ember applications. By taking advantage of new JavaScript features, tracked properties allow Ember to reduce its API surface area while producing code that is both more intuitive and less error-prone.
+
+During the Ember Octane preview period we encountered some issues with the current design for Tracked Properties that was proposed and accepted in [RFC 410](https://github.com/emberjs/rfcs/blob/master/text/0410-tracked-properties.md). The primary issues were specifically around interop between tracked properties, computed properties, and autotracking, with a few extra issues and inconsistencies surrounding these.
+
+With Ember.js 3.13 you now use this improved version of tracked properties. Have a look at the [guides] or [RFC 410](https://github.com/emberjs/rfcs/blob/master/text/0410-tracked-properties.md) to learn more about tracked properties. You can also have a look at [RFC 478](https://github.com/emberjs/rfcs/blob/master/text/0478-tracked-properties-updates.md) to see what the tracked properties update is about.
+
+**Component Templates Co-location (2 of 5)**
+
+With component templates co-location it is now possible to place a component's JavaScript file and it's template in the same directory on the file system.
+
+[The RFC](https://github.com/emberjs/rfcs/blob/master/text/0481-component-templates-co-location.md) contains a detailed explanaition for this feature.
```

Comment:
Only the low-level APIs: `setComponentTemplate()`, `getComponentTemplate()` and `templateOnlyComponent()` are enabled in this release. The supporting code for the actual co-location feature is included in the release, but is only enabled with octane apps... this feature will be enabled for all apps in the next version.

Reply:
@chancancode @kennethlarsen see my recommendation in the overall review comment I'm leaving; I'd suggest we note that users can try it out by testing Octane on the beta release channel.